internal/log.go

```diff
@@ -19,6 +22,20 @@ func (l *logger) Printf(ctx context.Context, format string, v ...interface{}) {
 	_ = l.log.Output(2, fmt.Sprintf(format, v...))
 }
 
+func (l *logger) Println(ctx context.Context, v ...interface{}) {
+	_ = l.log.Output(2, fmt.Sprintln(v...))
+}
+
+func (l *logger) Fatalf(ctx context.Context, format string, v ...interface{}) {
+	_ = l.log.Output(2, fmt.Sprintf(format, v...))
+	os.Exit(1)
+}
+
+func (l *logger) Fatalln(ctx context.Context, v ...interface{}) {
+	_ = l.log.Output(2, fmt.Sprintln(v...))
+	os.Exit(1)
+}
```

Comment:
Changes to the logging are unrelated and should be in a separate PR.

Reply:
Understand.